tests/test_0692_fsspec.py

```diff
@@ -0,0 +1,73 @@
+# BSD 3-Clause License; see https://github.com/scikit-hep/uproot4/blob/main/LICENSE
+
+import pytest
+
+import uproot
+import uproot.source.fsspec
+
+
+@pytest.mark.network
+def test_open_fsspec_http():
+    with uproot.open(
+        "https://github.com/CoffeaTeam/coffea/raw/master/tests/samples/nano_dy.root",
+        http_handler=uproot.source.fsspec.FSSpecSource,
+    ) as f:
+        data = f["Events/MET_pt"].array(library="np")
+        assert len(data) == 40
+
+
+@pytest.mark.network
+def test_open_fsspec_github():
+    pytest.skip("not working yet")
+    with uproot.open(
+        "github://CoffeaTeam:coffea@master/tests/samples/nano_dy.root",
+        http_handler=uproot.source.fsspec.FSSpecSource,
+    ) as f:
+        data = f["Events/MET_pt"].array(library="np")
+        assert len(data) == 40
+
+
+@pytest.mark.network
+def test_open_fsspec_local(tmp_path):
+    url = "https://github.com/CoffeaTeam/coffea/raw/master/tests/samples/nano_dy.root"
+
+    # download file to local
```

Comment:
Could this one be done with scikit-hep/testdata package instead?

Reply:
https://github.com/scikit-hep/scikit-hep-testdata/pull/130